g/github.com__vividcortex__ewma/github.com__vividcortex__ewma_rhel_8.3.sh

```diff
@@ -15,110 +15,54 @@
-		        exit 0
-		else		
-			echo "------------------$PACKAGE_NAME:install_&_test_both_success-------------------------"
-		        echo "$PACKAGE_VERSION $PACKAGE_NAME" > /home/tester/output/test_success
-		        echo "$PACKAGE_NAME  |  $PACKAGE_VERSION | master | $OS_NAME | GitHub  | Pass |  Both_Install_and_Test_Success" > /home/tester/output/version_tracker
-        		exit 0
-		fi
-	fi
-}
-
-function test_with_master(){
-	echo "Building $PACKAGE_PATH with master"
-	export GO111MODULE=auto
-	if ! go get -d -u -t $PACKAGE_NAME@$PACKAGE_VERSION; then
-		test_with_master_without_flag_u
-		exit 0
-	fi
-
-	cd $(ls -d $GOPATH/pkg/mod/$PACKAGE_NAME*)
-	echo "Testing $PACKAGE_PATH with $PACKAGE_VERSION"
-	# Ensure go.mod file exists
-	go mod init
-	if ! go test ./...; then
-		test_with_master_without_flag_u
-		exit 0
-	else
-		echo "------------------$PACKAGE_NAME:install_&_test_both_success-------------------------"
-		echo "$PACKAGE_VERSION $PACKAGE_NAME" > /home/tester/output/test_success 
-		echo "$PACKAGE_NAME  |  $PACKAGE_VERSION | $PACKAGE_VERSION | $OS_NAME | GitHub  | Pass |  Both_Install_and_Test_Success" > /home/tester/output/version_tracker
-		exit 0
-	fi
-}
-
-function test_without_flag_u(){
-	echo "Building $PACKAGE_PATH with $PACKAGE_VERSION and without -u flag"
-	if ! go get -d -t $PACKAGE_NAME@$PACKAGE_VERSION; then
-		test_with_master
-		exit 0
-	fi
+echo "Building $PACKAGE_PATH with $PACKAGE_VERSION"
 
-	cd $(ls -d $GOPATH/pkg/mod/$PACKAGE_NAME*)
-	echo "Testing $PACKAGE_PATH with $PACKAGE_VERSION"
-	# Ensure go.mod file exists
-	go mod init
-	if ! go test ./...; then
-		test_with_master
-		exit 0
-	else
-		echo "------------------$PACKAGE_NAME:install_&_test_both_success-------------------------"
-		echo "$PACKAGE_VERSION $PACKAGE_NAME" > /home/tester/output/test_success 
-		echo "$PACKAGE_NAME  |  $PACKAGE_VERSION | $PACKAGE_VERSION | $OS_NAME | GitHub  | Pass |  Both_Install_and_Test_Success" > /home/tester/output/version_tracker
-		exit 0
+if [ "$PACKAGE_VERSION" = "v1.2.0" ]; then
+	echo "VIKAS version is v1.2.0"
+	git clone --recurse https://github.com/vividcortex/ewma
+	git checkout $PACKAGE_VERSION
+	cd $PACKAGE_NAME
+else
+	echo "VIKAS version is v1.1.1"
+	if ! go get -d -u -t $PACKAGE_PATH@$PACKAGE_VERSION; then
+	        echo "------------------$PACKAGE_NAME:install_ failed-------------------------"
+        	exit 0
 	fi
-}
-
-echo "Building $PACKAGE_PATH with $PACKAGE_VERSION"
-if ! go get -d -u -t $PACKAGE_NAME@$PACKAGE_VERSION; then
-	test_without_flag_u
-	exit 0
+	cd $(ls -d $GOPATH/pkg/mod/$PACKAGE_PATH@$PACKAGE_VERSION)
 fi
 
-cd $(ls -d $GOPATH/pkg/mod/$PACKAGE_NAME*)
+echo `pwd`
+
 echo "Testing $PACKAGE_PATH with $PACKAGE_VERSION"
+
 # Ensure go.mod file exists
-go mod init
-if ! go test ./...; then
-	test_with_master
-	exit 0
+go mod init github.com/vividcortex/ewma
+go mod tidy
```

Comment:
This seems unnecessary for `v1.2.0` which is cloned and already has `go.mod`.

Reply:
Corrected the code for v1.2.0